src/commands/config.ts

```diff
@@ -75,14 +80,13 @@ const configPath = pathJoin(homedir(), '.opencommit');
 
 export const getConfig = (): ConfigType | null => {
   const configExists = existsSync(configPath);
-  if (!configExists) return null;
 
-  const configFile = readFileSync(configPath, 'utf8');
-  const config = iniParse(configFile);
+  const configFile = configExists ? readFileSync(configPath, 'utf8') : '';
+  const config = configFile? iniParse(configFile) : {};
 
-  for (const configKey of Object.keys(config)) {
+  for (const configKey in CONFIG_KEYS) {
     const validValue = configValidators[configKey as CONFIG_KEYS](
-      config[configKey]
+      config.hasOwnProperty(configKey) ? config[configKey] : process.env[configKey]
```

Comment:
Wouldn't you want the inverse of this setting the .env file as the priority and not the global config as priority?

Usually I think local config then fallback to global. I can also see just wanting to force the usage of one or the other via a process.env flag too.

Reply:
i dont think the inverse helps more here. you set the .env if you like it, it will work through .env. If you once set global config, you dont need env. works vice-versa

Reply:
Just found this pull request and would like to add my 2cnt.

I would prefer that the API Key from `.env` is used if set, because there might be cases where it differs from project to project.

- Project A could be for myself, where I pay for the usage (using global env)
- Project B could be a client project that's paid by _the company_ (using `.env`)
- Project C could be for open source project that's being paid by _some other entity_  (using `.env`)

Preferring project `.env` over global env allows these situations.